Podfile

```diff
@@ -113,38 +111,36 @@ target 'WordPress' do
     pod 'MRProgress', '0.8.3'
     pod 'Reachability',    '3.2'
     pod 'SVProgressHUD', '2.2.5'
-    pod 'Crashlytics', '3.11.0'
+    pod 'Crashlytics', '3.12.0'
     pod 'BuddyBuildSDK', '1.0.17', :configurations => ['Release-Alpha']
     pod 'Gifu', '3.2.0'
     pod 'GiphyCoreSDK', '~> 1.4.0'
-    pod 'MGSwipeTableCell', '1.6.7'
-    pod 'lottie-ios', '2.5.0'
+    pod 'MGSwipeTableCell', '1.6.8'
     pod 'Starscream', '3.0.6'
     pod 'ZendeskSDK', '2.2.0'
 
 
     ## Automattic libraries
     ## ====================
     ##
-    pod 'Automattic-Tracks-iOS', :git => 'https://github.com/Automattic/Automattic-Tracks-iOS.git', :tag => '0.2.3'
-    pod 'Gridicons', '0.16'
```

Comment:
This one I'm less sure of – it's used by multiple dependent libraries, each with their own versions. To avoid needing to update this every time a dependency bumps their version, I was thinking it made sense to just let the dependencies decide which version to show. 

This is one of our pods, and it only had additive changes, so I'm not particularly concerned about breaking changes. That said, we could ship a `1.0` of `Gridicons` to ensure we're sticking to SemVer, then pin this to something in the `~> 1.0.0` range?

WDYT?

Reply:
If its unused by WordPress itself I think it makes sense to get rid of it from here.

Reply:
So that's the tricky thing – it _is_ used directly in the app, _as well as_ in libraries. I want to avoid "DLL hell", which would argue for removing the pinned dependency, but I also don't want future breakage if we were to remove all the packages that depend on `Gridicons` (unlikely? also an easy fix?)

Reply:
Well if it is used directly in the app it should be in the `Podfile`.

Your previous suggestion of something like `~> 1.0.0` would work (using `~> 0.16` should be fine too).

Reply:
Fixed in 
https://github.com/wordpress-mobile/WordPress-iOS/pull/10827/commits/3f54a062a1892f4e44a4dfc034d84fd8e2c43bf8